webviews/components/sidebar.tsx

```diff
@@ -199,7 +199,27 @@ export default function Sidebar({ reviewers, labels, hasWritePermission, isIssue
 				{milestone ? (
 					<Milestone key={milestone.title} {...milestone} canDelete={hasWritePermission} />
 				) : (
-					<div className="section-placeholder">No milestone</div>
+					<>
+						<div className="section-placeholder">No milestone</div>
+					</>
+				)}
+			</div>
+			<div id="closingIssues" className="section">
+				<div className="section-header">
+					<div className="section-title">Linked Issues</div>
+				</div>
+				{closingIssues.length > 0 ? (
+					<div className="p-2">
```

Comment:
Please use a class name that is consistent with the class naming scheme in the file.

Reply:
Same for `gap-2` below.